package.json

```diff
@@ -46,18 +46,19 @@
     "lint": "samples lint",
     "generate": "node ./scripts/generate",
     "pretest": "npm run lint && node ./scripts/clean coverage",
-    "unit-test": "ava --verbose -T 30s functions/**/test/**/*.test.js",
```

Comment:
@jmdobry I removed the specific-file-tests in these commands. I assume those were included by mistake?